source/index.html.md

```diff
@@ -2656,7 +2632,7 @@ You can update the name, email, bank account and metadata of any Contact.
   <ul>
     <li>Previous transactions to this Contact will retain the name and bank account that was used at the time.</li>
     <li>You cannot update a Contact's bank account details if they currently have an accepted agreement.</li>
-    <li>See our [Help Article](https://help.split.cash/en/articles/3829211-how-do-i-change-my-customers-bank-account-details) for more information about the nuances and implications of changing a contacts Bank Account.</li>
+    <li>See our [Help Article](https://help.zepto.money/en/articles/6239641-nz-how-do-i-change-my-customers-bank-account-details) for more information about the nuances and implications of changing a contacts Bank Account.</li>
```

Comment:
It's still being created. Vishakha is in the process of writing the page, she just generated the link for me so I could include it here - but it will be available later once she has finished writing it